django/api/serializers/application_form.py

```diff
@@ -44,6 +45,13 @@ def create(self, validated_data):
         )
         return obj
 
+    def _getTaxYear(self):
```

Comment:
for Django functions should be named with snake case! so this should be get_tax_year

Reply:
fixed!